libs/python.json

```diff
@@ -133,5 +133,10 @@
   {"id":"py.gflags","imports":["gflags"],"name":"gflags","repo":"google/python-gflags","tags":["argument-parsing"],"tech":["utils"],"status":"model-ready"},
   {"id":"py.gooey","imports":["gooey"],"name":"gooey","repo":"chriskiehl/Gooey","tags":["cli2gui"],"tech":["utils"],"status":"model-ready"},
   {"id":"py.psutil","imports":["psutil"],"name":"psutil","repo":"giampaolo/psutil","tags":["system"],"tech":["utils"],"status":"awaiting-model"},
-  {"id":"py.python-dateutil","imports":["dateutil"],"name":"dateutil","repo":"dateutil/dateutil","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"}
+  {"id":"py.python-dateutil","imports":["dateutil"],"name":"dateutil","repo":"dateutil/dateutil","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"},
+  {"id":"py.tqdm","imports":["tqdm"],"name":"tqdm","repo":"tqdm/tqdm","tags":["process-runner"],"tech":["cli"],"status":"awaiting-model"},
+  {"id":"py.paramiko","imports":["paramiko"],"name":"paramiko","repo":"paramiko/paramiko","tags":["ssh-lib"],"tech":["networking"],"status":"awaiting-model"},
```

Comment:
just "ssh". everything is "-lib" here

Reply:
we used to have ssh-lib in networking, cs.ssh-net in particular. I'll change it as well then